docs/acp/user/intro-to-acp/acp-installation.md

```diff
@@ -0,0 +1,485 @@
+
+```json
+#AOP
+SPRYKER_AOP_APPLICATION: '{
+  "APP_CATALOG_SCRIPT_URL": "https://app-catalog.atrs.spryker.com/loader",
+  "STORE_NAME_REFERENCE_MAP": {"DE": "tenant_messages_for_store_reference_AOP_Demo_Production-DE.fifo", "AT": "tenant_messages_for_store_reference_AOP_Demo_Production-AT.fifo"},
+  "APP_DOMAINS": [
+      "os.apps.aop.spryker.com",
+      "*.bazaarvoice.com"
+  ]
+}'
+
+SPRYKER_AOP_INFRASTRUCTURE: '{
+  "SPRYKER_MESSAGE_BROKER_HTTP_SENDER_CONFIG": {
+        "endpoint":"https:\/\/events.atrs.spryker.com\/events\/tenant"
+  },
+  "SPRYKER_MESSAGE_BROKER_SQS_RECEIVER_CONFIG": {
+    "default": {
+        "endpoint":"https:\/\/sqs.eu-central-1.amazonaws.com",
+        "region":"eu-central-1",
+        "auto_setup":false,
+        "buffer_size":1
+    },
+      "DE": {
+        "queue_name":"tenant_messages_for_store_reference_AOP_Demo_Production-DE.fifo"
+    },
+    "AT": {
+        "queue_name":"tenant_messages_for_store_reference_AOP_Demo_Production-AT.fifo"
+    }
+  }
+}'
+```
+</details>
+
+#### General configurations
+
+**Variable name**: `SPRYKER_AOP_APPLICATION`
+
+**1. Configuration key**: `APP_CATALOG_SCRIPT_URL`
+
+**Explanation**: URL for the App-Tenant-Registry-Service (ATRS) and path to the JS script to load the ACP catalog
+
+<details open>
+<summary>Value: Production ATRS_HOST and path</summary>
+
+```json
+https://app-catalog.atrs.spryker.com/loader
+```
+</details>
+
+**2. Configuration key**: `STORE_NAME_REFERENCE_MAP`
+
+**Explanation**: StoreReference mapping to the appropriate Spryker Store
+
+**NOTE**: StoreReference mapping will be created by Spryker Ops and values provided by them for you to add!
+
+<details open>
+<summary>Value Example: Demo Stores for DE and AT</summary>
+
+```json
+{"DE": "tenant_messages_for_store_reference_AOP_Demo_Production-DE", "AT": "tenant_messages_for_store_reference_AOP_Demo_Production-AT"}
+```
+</details>
+
+**3. Configuration key**: `APP_DOMAINS`
+
+**Explanation**: App domains used in redirects.
+
+**NOTE**: Value must be provided by Spryker Ops!
+
+<details open>
+<summary>Value Example</summary>
+
+```json
+[
+  "os.apps.aop.spryker.com",
+  "*.bazaarvoice.com"
+]
+```
+</details>
+
+#### Message Broker configuration
+
+**Variable name**: `SPRYKER_AOP_INFRASTRUCTURE`
+
+**1. Configuration key**: `SPRYKER_MESSAGE_BROKER_SQS_RECEIVER_CONFIG`
+
+**Explanation**: Receiver configuration. The queues must be defined for each store (or default queue for all stores is used)
+
+**NOTE**: Queues will be created by Spryker Ops and values provided by them for you to add!
+
+<details open>
+<summary>Value Example: Spryker Production Environment</summary>
+
+```json
+{
+  "default": {
+    "endpoint":"https://sqs.eu-central-1.amazonaws.com",
+    "auto_setup":false, "buffer_size":1
+  },
+  "DE": {
+    "queue_name":"tenant_messages_for_store_reference_AOP_Demo_Production-DE.fifo"
+  },
+  "<Store Reference>": {
```

Comment:
It is not a `Store reference` it is Store. store reference is smth like this: `tenant_messages_for_store_reference_AOP_Demo_Production-DE`

Reply:
Corrected that in the internal documentation where I took the example from and here was well.